src/miner.cpp

```diff
@@ -917,12 +917,14 @@ void ThreadStakeMiner(CWallet *pwallet, CConnman* connman)
         CAmount nTargetValue = nBalance - pwallet->m_reserve_balance;
         CAmount nValueIn = 0;
         std::set<std::pair<const CWalletTx*,unsigned int> > setCoins;
+        std::set<std::pair<const CWalletTx*,unsigned int> > setDelegateCoins;
         int64_t start = GetAdjustedTime();
         {
             auto locked_chain = pwallet->chain().lock();
             pwallet->SelectCoinsForStaking(*locked_chain, nTargetValue, setCoins, nValueIn);
```

Comment:
Addresses in this wallet which are delegated to others should not be included in `setCoins`

Reply:
This is related or present in the hybrid staking issues described by caspal. @qtum-neil Does modification is required?